src/DynamoUtilities/Md2Html.cs

```diff
@@ -76,6 +79,37 @@ internal string ParseMd2Html(string mdString, string mdPath)
             return GetData(processCommunicationTimeoutms);
         }
 
+        private void EncodeMDContent(ref string mdContent)
+        {
+            //Encode to base64 due that the Tools / Md2Html console app is using a different encoding and special characters are lost when sending info to Stdio
+
+            //Get the MD file content and encode it
+            EncodeBase64(ref mdContent, @"#+\s[^\n]*\n(.*?)(?=\n##?\s|$)");
+
+            //Get the MD file headers and encode it
+            EncodeBase64(ref mdContent, @"#+\s(.*?\n)");
+        }
+
+        private void EncodeBase64(ref string mdContent, string regEx)
```

Comment:
Would you put some comments to the function? And do you think we can add a test case?

Reply:
Added comments for the method and the parameters received.
Related to the test case, I think there is no way to check if the content of the DocumentationBrowser is showing the expected characters in a specific language, probably something that we can do is trying to process some md files in different languages and then check that the generated html has a similar content that the md file.
What do you think?
https://github.com/DynamoDS/Dynamo/pull/15377/commits/33430d9cb836caa94d02429978d28021579d2319

Reply:
@RobertGlobant20 yup, totally fine. No need to switch Dynamo to a different locale, loading md file in a different locale and check content is a fine test case

Reply:
I've added a unit test that validates that the extended characters (in en-US, ko-KR, cs-CZ) are processed correctly by the Md2Html.exe tool.
https://github.com/DynamoDS/Dynamo/pull/15377/commits/1d7cd8d0afae6f3efa75f9e50509c28ae88ad8e5